packages/loopring_v3/contracts/iface/IExchangeV3.sol

```diff
@@ -64,62 +48,45 @@ abstract contract IExchangeV3 is IExchange
         address         newOperator
     );
 
-    event AddressWhitelistChanged(
-        uint    indexed exchangeId,
-        address         oldAddressWhitelist,
-        address         newAddressWhitelist
-    );
-
-    event FeesUpdated(
-        uint    indexed exchangeId,
-        uint            accountCreationFeeETH,
-        uint            accountUpdateFeeETH,
-        uint            depositFeeETH,
-        uint            withdrawalFeeETH
+    event Shutdown(
```

Comment:
maybe we also want to provide a view function `getShutdownStartTime`

Reply:
There is `isShutdown`, and this event, not sure how helpful this function would be. Doesn't seem super useful to know onchain.

Reply:
Not saying it won't be useful, but it won't be useful onchain I'd think. Are users going to to go the contract to query exchange data there? Pretty much nothing is stored onchain so they have to depend on some UI for much more important things. People have staked millions of dollars and they aren't even bothered to call the function on the smart contract to actually withdraw the protocol fees... And if they really want they may as well just search for the event and find it like that on etherscan, they're already there anyway.